File: javascript/logic.js
// no goto statements in js :(
l();
function n () {
  console.log('is');
  h();
}

function l () {
  console.log('this');
  y();
}

function h () {
  console.log('broken');
}

function y () {
  console.log('logic');
  n();
}

File: c/logic.c
#include <stdio.h>

int main(int argc, char **argv)
{
    goto l;
n:  printf("is ");
    goto h;
l:  printf("this ");
    goto y;
h:  printf("broken\n");
    return 0;
y:  printf("logic ");
    goto n;
}
